content/cookbook/accounts/close-account.md

```diff
@@ -0,0 +1,45 @@
+---
+title: How to Close an Account
+sidebarSortOrder: 5
+description:
+  "When an account is no longer needed, you can close the account to reclaim the
+  rent. Learn how to close accounts efficiently on Solana."
+---
+
+Closing accounts enables you to reclaim the SOL that was used to open the
+account, but requires deleting of all information in the account. When an
+account is closed, make sure that the data is zeroed out in the same instruction
```

Comment:
why does it matter to zero out the data while closing the account?

Reply:
Security issue. Account is not garbage collected until after the transaction finished executing. There's the possibility for an account to be closed in an instruction and reopened in a later instruction under the same transaction, utilizing values that were previously there. Can also open up to a whole array of attacks.